packages/webdriverio/src/utils/index.ts

```diff
@@ -101,11 +101,11 @@ export function getBrowserObject (elem: WebdriverIO.Element | WebdriverIO.Browse
 /**
  * transform whatever value is into an array of char strings
  */
-export function transformToCharString (value: any, translateToUnicode = true) {
+export function toUnicodeCharacterArray (value: string | number | (string|number)[], translateToUnicode = true) {
     const ret: string[] = []
 
     if (!Array.isArray(value)) {
-        value = [value]
+        value = [value as (number | number)]
```

Comment:
seems to be a duplicate
```suggestion
        value = [value as string | number]
```

Reply:
this is actually correct it's either string or number and needs to be surrounded by () or it won't accept it correctly

Reply:
@erwinheitzman what you propose is `value = [value as (number | number)]` which seems not right? It says cast it to number or number?

Reply:
The value can be a string, number or an array containing numbers and/or strings. If the value is not already an array, which is needed in this case, we wrap the value in an array. In order to do that you need to tell the compiler that the type of the value that we want to wrap is of type string or number instead of type Array<string | number> because this is the type we want to end up with.

All this does is: if the value isn't already an array (thus either a number or string), wrap it in an array (which can also contain strings and/or numbers).

Reply:
I'm going to revert this and add a warning like you suggested though.

Although I can't imagine anyone doing things like `setValue([1, '2', [1, '2'], {}, true])` and such because you'd just end up with an output of `'12[12]{}true'`